### ISSUE TITLE
How do I set the public path aka basename?

### ISSUE THREAD

**Issue by Pyrolistical** [hyperapp/router#55]
On my local, I use http://localhost:8080/greetings, but deployed its https://production/some-path/greetings

In my code, I want to write `<Route path="/greetings" render={Greetings} />`

React-router has this:
https://reacttraining.com/react-router/web/api/BrowserRouter/basename-string

**Comment by jorgebucaran** [comment]
Let's basename then. 👍

**Comment by Swizz** [comment]
I agree. I needed it in some projects. And the concatenation trick is not really useful.
